src/configure/configure.ts

```diff
@@ -173,8 +182,12 @@ class Orchestrator {
                 this.inputs.azureSession = getAzureSession();
             }
             const repoAnalysisResult = await this.getRepositoryAnalysis();
-            await this.getSelectedPipeline(repoAnalysisResult);
-
+            await this.setPipelineType();
+            // Right click on not resource not supported for Azure pipeline
+            if (resourceNode && this.pipelineType === PipelineType.AzurePipeline) {
+                throw Error("Scenario not supported for Azure pipelines.");
```

Comment:
Do we need to explicitly throw this error? Anyways getTemplatesByRepoAnalysis will return empty array of templates and hence a n error will be shown to user

Reply:
If the repo has something besides AKS, it will cause an issue and the flow will try to continue with webApp template and will break. So, anyway, we know that this scenario is not supported by us. therefore, explictly throwing. Better to have a proper error message to use instead of some dev exception.

Reply:
Okay, so after reading through and thinking a lot I think this is okay. I can justify this explicit condition so that we do not have to start filtering local templates by target resource type.